src/lib/parserRaw.c

```diff
@@ -9,6 +9,8 @@
 #include "../../deps/redis/util.h"
 #include "../../deps/redis/listpack.h"
 #include "../../deps/redis/ziplist.h"
+#include "../../deps/redis/zipmap.h"
+#include "../../deps/redis/intset.h"
```

Comment:
unrelated to these lines.. maybe it's a good idea to add a comment at the top of each of the main files explaining their role. i.e. parser.c vs parserRaw.c